.gitattributes

```diff
@@ -1,5 +1,6 @@
 /.github/                       export-ignore
 /.phive/                        export-ignore
+/src/Test/                      export-ignore
```

Comment:
When you work on a package and declare test code in a subdirectory of the production code, adding exclusions for test code in `.gitattributes` is less work.